routers/web/web.go

```diff
@@ -1181,6 +1181,7 @@ func registerRoutes(m *web.Route) {
 			})
 
 			m.Post("/labels", reqRepoIssuesOrPullsWriter, repo.UpdateIssueLabel)
+			m.Post("/labels2", reqRepoIssuesOrPullsWriter, repo.UpdateIssueLabelBatched)
```

Comment:
This is not a good name.

Reply:
Maybe `/labels/batch` or `/batch_labels` etc.

Reply:
either is fine, used /batch_labels